kolibri/plugins/learn/assets/src/views/EmbeddedSidePanel/index.vue

```diff
@@ -305,6 +300,10 @@
       noCategories() {
         this.$emit('input', { ...this.value, categories: { [NoCategories]: true } });
       },
+      hasNestedCategories(key) {
```

Comment:
If this is `category` instead of `key` as I suggest above, this function can just be:
```
return Object.keys(libraryCategories[category].nested).length > 0;
```